api/client/webclient/webclient.go

```diff
@@ -90,6 +94,53 @@ func Ping(ctx context.Context, proxyAddr string, insecure bool, pool *x509.CertP
 	return pr, nil
 }
 
+// GetTunnelAddr returns the tunnel address retrieved from the web proxy.
+func GetTunnelAddr(ctx context.Context, proxyAddr string, insecure bool, pool *x509.CertPool) (string, error) {
```

Comment:
Why does the `pool` parameter exist? I can't see it used, and I can only see `nil` passed to it.

Reply:
The other webclient endpoints accept a pool parameter, so I think this one should too. But to be honest I'm not sure why it exists on those either as I don't think it's used.